src/Applications/LDAS_App/ldas_setup

```diff
@@ -51,11 +51,11 @@ class LDASsetup:
         # ------
         # Required resource manager input fields
         # ------
-        rqdRmInpKeys = ['rm_name', 'account', 'walltime', 'ntasks']
+        rqdRmInpKeys = ['account', 'walltime', 'ntasks_model', 'ntasks-per-node']
```

Comment:
Why drop "rm_name"?  So far we only support SLURM, but if we ever need to support another rm, it'll be easier to make the changes if we already have a template for handling "rm_name".  Perhaps this somehow handled in the new MAPL or I'm missing something else, but I thought I should ask.

Reply:
SLURM is quite universal now.  I think  we have already dropped off the other system (like PBS) since we don't have it on NCCS. Without testing, it can go wrong easily.

Reply:
It's ok to drop.